### DISCUSSION TITLE
Row Count exceeded 100000

### DISCUSSION THREAD

**Discussion by SriramItrend** [planetscale/discussion#579] [Q&A]
I am using planetscale - Mysql Server 
if trying to add more than 1,00,000 rows in my table , throw the error

"
Mysql:target:itrenddb.-.primary:vttablet:rpc error :code = Aborted desc = Row Count exceeded 100000 (CallerID: 1vcsx0eo349iqhfcw4c0)

"

Please be suggest how to solve

**Comment by orware** [comment]
Hello @SriramItrend,

Is the query involved here a `SELECT` query or some type of `INSERT` / `UPDATE` / `DELETE` query?

We do have our 100K row limit documented here:
https://planetscale.com/docs/reference/planetscale-system-limits

But typically the workaround needed is to use a `LIMIT` / `OFFSET`, or pagination options that could be built-in to your ORM if you are using one, so that you can retrieve or operate on a subset of rows at a time (e.g. 100 / 1000 / 10000) until you work through the entire result set.

**Comment by SriramItrend** [comment]
Hi Omar ,

      If I try to select the 1,00,000 above table values I get an error ( Row
Count exceeded 100000 ) .

Regards,
Sriram Jayavel
